core/src/main/java/com/megaease/easeagent/core/plugin/CommonInlineAdvice.java

```diff
@@ -46,6 +48,7 @@ public static MethodInfo enter(@Index int index,
         if (context.isNoop()) {
             return null;
         }
+        System.out.printf("index: %d, invoker: %s, type: %s, method: %s, all arguments: %s, context: %s\n", index, invoker, type, method, Arrays.toString(args), context);
```

Comment:
Please ensure the cleanliness of your code, do not put the test code in the execution code

Reply:
I have cleaned this line of code.